FEM/fem_ele_std.cpp

```diff
@@ -9004,6 +9970,26 @@ void CFiniteElementStd::Assembly()
 #endif
 			break;
 		//....................................................................
+        case EPT_HEAT_TRANSPORT_BHE: 
+            heat_phase_change = false; // ?2WW
+
+            if (this->pcs->getProcessType() == FiniteElement::HEAT_TRANSPORT_BHE && ele_dim == 1)
+            {
+                // this is a BHE element
+                AssembleMixedHyperbolicParabolicEquation_BHE();
+            }
+            else
+            {
+                //CMCD4213
+                AssembleMixedHyperbolicParabolicEquation();
```

Comment:
Yes, I do. At first glance this is the same as the HEAT_TRANSPORT process. Actually it is NOT. With the BHE feature, both assembly process from the original HEAT_TRANSPORT and the newly added BHE part are executed. This structure makes it happen while keeping the original HEAT_TRANSPORT unchanged.